benchmarks/serde-benchmarks-codegen/src/main/kotlin/SerdeBenchmarkJsonProtocol.kt

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+package software.amazon.smithy.kotlin.codegen.protocols
+
+import software.amazon.smithy.kotlin.codegen.integration.KotlinIntegration
+import software.amazon.smithy.kotlin.codegen.rendering.protocol.ProtocolGenerator
+import software.amazon.smithy.model.shapes.ShapeId
+
+/**
+ * Dummy protocol for use in serde-benchmark project models. Generates JSON based serializers/deserializers
```

Comment:
nit

Maybe there is something conveyed in the term 'dummy' but 'test protocol' sounds more descriptive to me.